public/blog/_data.json

```diff
@@ -1,4 +1,9 @@
 {
+  "crosswalk-app-tools-07": {
+    "title": "Crosswalk App Tools, the easiest way to package Crosswalk Applications",
```

Comment:
What about "The fastest way to package Web Applications" (but yours is good too)